src/webui/api/torrentscontroller.cpp

```diff
@@ -682,6 +683,37 @@ void TorrentsController::removeTrackersAction()
         torrent->forceReannounce();
 }
 
+void TorrentsController::addPeersAction()
+{
+    checkParams({"hash", "peers"});
+
+    const QString hash = params()["hash"];
```

Comment:
Don't you want to make this action to perform a batch? I.e. accepting multiple hashes here?